### ISSUE TITLE
Alexa's voice reply works, but can't hear music or flash briefings

### ISSUE THREAD

**Issue by hydr0boy** [alexa/alexa-avs-sample-app#1181]
Hello,

I'll be the first to admit I may be slightly outside of this groups designated subject matter.

I've got alexa working perfectly well, she plays music and everything just fine, Thanks!

I'm also trying to incorporate a 'sensehat' into the build. My dream is for it to pick up everything that the mic does and visualise it on it's LED matrix. 

Using this:
https://github.com/rubenmak/raspberry-pi-sense-hat-audio-spectrum-analyzer

**I found that by editing the:
/usr/share/alsa/alsa.conf
and turning the "defalts.pcm.card" from a 0 to a 1 I can get the visualiser to work.  

Doing this edit, however, breaks something in the audioplay back in Alexa. For example: it plays Alexa's voice and responses to to weather and jokes etc just fine, but it can't play external audio sources like iHeart Radio or Flash briefings.
 
TL;DR- n00b found a way to break Alexa, trying to run a third party app, and came on here ask for help

**Comment by hydr0boy** [comment]
I should add: using a PlayStation eye cam mic, and the 3.5 mm audio out.

**Comment by frozenberg** [comment]
Hi @hydr0boy,

What does your `~/.asoundrc` file look like? It's possible that flipping that defaults.pcm.card value can screw up your audio configuration.

Before going too much further I would also advice switching over to the C++ SDK. You can find quick start guides for rPi and Linux here: https://github.com/alexa/avs-device-sdk

The C++ SDK is better supported and not on deprecation path. If there was a bug with our audio playback of external sources, it's highly likely that we've addressed it in the C++ SDK.

Fede
